downloader/src/funTest/kotlin/vcs/GitRepoDownloadTest.kt

```diff
@@ -77,7 +76,11 @@ class GitRepoDownloadTest : StringSpec() {
                 "test-data-npm"
             )
 
-            val actualSubmodulesFiles = submodulesDir.listFiles(FileFilter { it.isDirectory }).map { it.name }.sorted()
+            val actualSubmodulesFiles = submodulesDir.walk().maxDepth(1).filter {
+                it.isDirectory
+            }.map {
+                it.name
+            }.sorted()
```

Comment:
The similar code takes one line above and five lines here, maybe consider reformatting it to
```
val actualSubmodulesFiles = 
    submoduelsDir...
```
or
```
val actualSubmodulesFiles = submodulesDir.walk().maxDepth(1)
    .filter { it.isDirectory }
    .map { it.name }
    .sorted()
```

Reply:
TBH, I don't think it's worth addressing that now that the PR already has passed (and you didn't explicitly request changes via a review).